src/components/common/YourCompany/YourCompany.vue

```diff
@@ -178,6 +178,8 @@
       </v-row>
     </div>
 
+    <v-divider v-if="isChangeFiling" class="mx-4 my-1" />
```

Comment:
Or alteration filing?

Reply:
Oddly enough this divider is exclusive to Change Filings. 
I asked directly about this because i missed it in the initial design and was part of UX feedback. 
But yea, unique design for this filing type 😝